src/read.c

```diff
@@ -4712,6 +4730,14 @@ avifResult avifDecoderReset(avifDecoder * decoder)
         }
 #endif // AVIF_ENABLE_EXPERIMENTAL_GAIN_MAP
 
+        // When searching for alpha and gainmap items, it is possible that the data->meta->items array is re-sized in calls to
+        // avifMetaFindOrCreateItem. So it is not safe to store pointers into data->meta->items array until all the items are
```

Comment:
Nit: add "the" before "data->meta->items array"

I am sorry I misunderstood what "that array" referred to. You can change this back to "that array".

Reply:
I added `the` and left it as data->meta->items.